ParisBNF/abb/BNFabb97.xml

```diff
@@ -0,0 +1,1357 @@
+              <msItem xml:id="ms_i1.109">
+                <title type="complete" ref="LIT1709Kebran#Chapter109" xml:lang="gez">በእንተ፡
+                  ስቅለቱ፡</title>
+                <textLang mainLang="gez"/>
+              </msItem>
+
+              <msItem xml:id="ms_i1.110">
+                <title type="complete" ref="LIT1709Kebran#Chapter110" xml:lang="gez">በእንተ፡
+                  ትንሣኤሁ፡</title>
+                <textLang mainLang="gez"/>
+              </msItem>
+
+              <msItem xml:id="ms_i1.111">
+                <title type="complete" ref="LIT1709Kebran#Chapter111" xml:lang="gez">በእንተ፡ ዕርገቱ፡
+                  ወዳግም፡ ምጽአቱ፡</title>
+                <textLang mainLang="gez"/>
+              </msItem>
+
+              <msItem xml:id="ms_i1.112">
+                <title type="complete" ref="LIT1709Kebran#Chapter112" xml:lang="gez">በእንተ፡ አርአዩ፡
+                  ነቢያት፡</title>
+                <textLang mainLang="gez"/>
+              </msItem>
+
+              <msItem xml:id="ms_i1.113">
+                <title type="complete" ref="LIT1709Kebran#Chapter113" xml:lang="gez">በእንተ፡ ሠረገላ፡
+                  ወመግረሬ፡ ፀር፡</title>
+                <textLang mainLang="gez"/>
+              </msItem>
+
+              <msItem xml:id="ms_i1.114">
+                <title type="complete" ref="LIT1709Kebran#Chapter114" xml:lang="gez">በእንተ፡ ንጉሠ፡
+                  ኢትዮጵያ፡</title>
+                <textLang mainLang="gez"/>
+              </msItem>
+
+              <msItem xml:id="ms_i1.115">
+                <title type="complete" ref="LIT1709Kebran#Chapter115" xml:lang="gez">በእንተ፡ ግብዓታ፡
+                  ለጽዮን፡</title>
+                <textLang mainLang="gez"/>
+              </msItem>
+
+              <msItem xml:id="ms_i1.116">
+                <title type="complete" ref="LIT1709Kebran#Chapter116" xml:lang="gez"> በእንተ፡ ቅስተ፡
+                  እስራኤል፡</title>
+                <textLang mainLang="gez"/>
+              </msItem>
+
+              <msItem xml:id="ms_i1.117">
+                <title type="complete" ref="LIT1709Kebran#Chapter117" xml:lang="gez">በእንተ፡ ሰረገላ፡
+                  ኢትዮጵያ፡</title>
+                <textLang mainLang="gez"/>
+              </msItem>
+              <msItem xml:id="ms_i1.118">
+                <title type="complete" ref="LIT1709Kebran#Chapter118" xml:lang="gez">በእንተ፡ ንጉሠ፡ ሮሜ፡
+                  ወንጉሠ፡ ኢትዮጵያ፡</title>
+                <textLang mainLang="gez"/>
+              </msItem>
+              <colophon xml:id="coloph1" xml:lang="gez"><locus target="#81"/> ዛቲሰ፡ ንግሥተ፡ አዜብ፡ እምድኅረ፡
+                ተመይጠት፡ እምሰሎሞን፡ ንጉሥ፡ ፳ወ፭፡ ዓመተ፡ ኀበ፡ ሀገረ፡ አክሱም። ወኵሉ፡ መዋዕለ፡ መንግሥታ፡ ድሙር፡ ፸ወ፭፡ ዓመት፡ ነገሥት፡
+                በሰላም፡ </colophon>
+              <colophon xml:id="coloph2" xml:base="gez">
+                <locus from="81rb" to="81vb"/>
+                <unclear/> መጽሐፍ፡ አውፃእናሃ ፡ እመጽሐፈ ፡ ቅብጢ ፡ ለዓረቢ ፡ እመቃብረ ፡ ማርቆስ ፡ ወንጌላዊ ፡ መምህር ፡ አበ ፡
+                ኵልነ ፡ ወአውፃእናሃ ፡ በ፬፻ ወ፱ ዓመተ ፡ ምሕረት ፡ ውስተ ፡ ሀገረ ፡ ኢትዮጵያ ፡ በመዋዕለ ፡ ገብረ ፡ መስቀል ፡ ንጉሥ ፡
```

Comment:
In this colophon, there are space before the word dividers. If it is not intentional, I suggest to delete the spaces the conform to the general bm principle